### PR TITLE
fix: Improve a11y for anonymous consents dialog

### PR DESCRIPTION
**Pull request by sdrozdsap**
* Display notification when giving/withdrawing a consent
* Clear All/Allow All buttons are accessible from the keyboard

### CODE REVIEW

File: projects/storefrontlib/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.ts [hunk 1]
```diff
@@ -177,6 +191,28 @@ export class AnonymousConsentDialogComponent implements OnInit, OnDestroy {
     return null;
   }
 
+  private onConsentGivenSuccess(): void {
```
Comment (Zeyber) [inline projects/storefrontlib/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.ts]: Please use `protected` modifiers for methods if they should not be public-facing. `private` is unaccessible to the public interface at all which requires users to copy spartacus libraries in order to make changes if needed.

File: projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.html [hunk 2]
```diff
@@ -1,40 +1,101 @@
...
+      loading: loading$ | async,
+      templateList: templateList$ | async,
+      allConsentsLoading: allConsentsLoading$ | async
+    } as data"
+  >
+    <ng-container *ngIf="data.templateList as templateList">
+      <div class="cx-consent-action-links">
+        <div class="col-sm-12 col-md-8 col-lg-6">
+          <button
+            tabindex="0"
```
Comment (Zeyber) [inline projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.html]: We can remove these `tabindex`s from `button` elements as they should be supported natively.

File: projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts [hunk 3]
```diff
@@ -42,9 +42,13 @@ export class ConsentManagementComponent implements OnInit, OnDestroy {
 
   templateList$: Observable<ConsentTemplate[]>;
   loading$: Observable<boolean>;
-
+  initialLoading$: Observable<boolean>;
```
Comment (Zeyber) [inline projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts]: Can we instantiate this variable here before `ngOnInit()`?

File: projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts [hunk 4]
```diff
@@ -181,6 +186,10 @@ export class ConsentManagementComponent implements OnInit, OnDestroy {
     );
   }
 
+  identifyT(_: number, tl: ConsentTemplate) {
```
Comment (Zeyber) [inline projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts]: What is this doing?

**Review by Zeyber — DISMISSED** [review]

File: projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts (hunk 4 above)
Comment (sdrozdsap) [reply projects/storefrontlib/cms-components/myaccount/consent-management/components/consent-management.component.ts]: Removed

**Review by rmch91 — APPROVED** [review]

**Review by developpeurweb — APPROVED** [review]

**Pull request merged**